index.js

```diff
@@ -96,11 +96,17 @@ module.exports = class ProtomuxRPC extends EventEmitter {
       try {
         if (requestEncoding) value = c.decode(requestEncoding, value)
 
-        value = await responder.handler(value)
+        try {
+          value = await responder.handler(value)
+        } catch (err) {
+          safetyCatch(err)
+
+          error = errors.REQUEST_ERROR('Request failed', err)
```

Comment:
I like this approach, just commenting to make it explicit that it is now the server's responsibility to not leak sensitive data in the err.message object, since it's sent to the client. It might be worth documenting that